src/index.js

```diff
@@ -1 +1,38 @@
-'use strict';
+// 'use strict';
+import express from 'express';
+import cookieParser from 'cookie-parser';
+import cors from 'cors';
+import { authRouter } from './routes/auth.route.js';
+import { userRoute } from './routes/user.route.js';
```

Comment:
There is an inconsistency in naming conventions between 'authRouter' and 'userRoute'. It would be better to use consistent naming conventions for imported routes, such as 'authRoute' and 'userRoute', to maintain uniformity.